R/dev-version.R

```diff
@@ -6,12 +6,7 @@
 #' @export
 use_dev_version <- function() {
   check_is_package("use_dev_version()")
-  if (uses_git() && git_uncommitted()) {
-    stop(
-      "Uncommited changes. Please commit to git before continuing",
-      call. = FALSE
-    )
-  }
+  check_uncommitted_changes(proj_get())
```

Comment:
Pretty sure this can just be `check_uncommitted_changes()`, right?

Reply:
Yes, I was not sure because it appeared to me that in some places the path was set while in other places it was not, so I thought I go the safe way and add it. Will remove it then.